microsoft/testsuites/mshv/mshv_root_stress_tests.py

```diff
@@ -120,7 +120,7 @@ def stress_mshv_vm_create(
                     test_status=TestStatus.FAILED,
                     test_message=repr(e),
                 )
-        self._save_dmesg_logs(node, log_path)
+        node.tools[CloudHypervisor].save_dmesg_logs(node, log_path)
```

Comment:
Why make this change? The dmesg tool should work with any platform.

Reply:
I figured since I was sharing the routine originally in the root_stress test, I'd put it in the CloudHypervisor tool (to me, this is the "common" code?)

Reply:
Ok, good to keep it.